src/js/components/stock-movement-wizard/inboundV2/sections/InboundSendForm.jsx

```diff
@@ -185,14 +199,29 @@ const InboundSendForm = ({ previous }) => {
             label="react.default.button.previous.label"
             defaultLabel="Previous"
             variant="primary"
-            onClick={previous}
-          />
-          <Button
-            label="shipping.sendShipment.label"
-            defaultLabel="Send shipment"
-            variant="primary"
-            type="submit"
+            onClick={previousPage}
+            disabled={statusCode === requisitionStatus.DISPATCHED}
           />
+          <div className="buttons-container">
+            {(hasAdminRights && shipped) && (
+              <Button
+                label="react.default.button.rollback.label"
+                defaultLabel="Rollback"
+                variant="primary-outline"
+                onClick={rollbackStockMovement}
```

Comment:
to be safer I will always suggest making it `onClick={() => rollbackStockMovement()}`